samples/core/blur/blur.cpp

```diff
@@ -527,7 +529,7 @@ void BlurCppExample::build_program(std::string options)
     // Open kernel stream if not already openned.
     if (!kernel_stream.is_open())
     {
-        const char* kernel_location = "./blur.cl";
+        const char* kernel_location = "C:/OpenCL/OpenCL-SDK/samples/core/blur/blur.cl";
```

Comment:
Not intended to be changed

Reply:
@Mikolaj-Filar-Mobica thanks for that- you are right it was added by mistake. Just removed back.